app/models/rules_of_origin/origin_reference_document.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
```

Comment:
You shouldn't need this - I suspect the wrong local rubocop config is being used